Adding a next steps page to the XR documentation (#9493)
godotengine/godot-docs: tutorials/xr/xr_next_steps.rst

.. _doc_xr_next_steps:

Where to go from here
=====================

> Now that we have the basics covered there are several options to lookat for your XR game dev journey:

Reviewer: ```suggestion
Now that we have the basics covered there are several options to look at for your XR game dev journey:
```

* You can take a look at the :ref:`Advanced topics <openxr-advanced-topics>` section.

> * You can look at a number of `XR demo's here <https://github.com/godotengine/godot-demo-projects/tree/master/xr>`_.

Reviewer: ```suggestion
* You can look at a number of `XR demos here <https://github.com/godotengine/godot-demo-projects/tree/master/xr>`_.
```
Plural and not possession

* You can find 3rd party tutorials on our :ref:`Tutorials and resources <doc_community_tutorials>` page.

XR Toolkits
-----------

There are various XR toolkits available that implement more complex XR logic ready for you to use.

> We have a :Ref:`small introduction into Godot XR Tools <godot-xr-tools>` that you can look at,

Reviewer: ```suggestion
We have a :Ref:`small introduction to Godot XR Tools <godot-xr-tools>` that you can look at,
```
Into indicates direction
